templates/pages/consent.html

```diff
@@ -40,9 +40,13 @@ <h1>wants to access your Matrix account</h1>
                   {% for scope in grant.scope | split(pat=" ") %}
                     {% if scope == "openid" %}
                       <li>See your profile info and contact details</li>
-                    {% elif scope is matching("^urn:matrix:org.matrix.msc2967.client:device:") %}
+                    {% elif scope == "urn:matrix:org.matrix.msc2967.client:api:*" %}
                       <li>View your existing messages and data</li>
                       <li>Send new messages on your behalf</li>
+                    {% elif scope == "urn:synapse:admin:*" %}
+                      <li>Administer the Synapse homeserver</li>
+                    {% elif scope is matching("^urn:matrix:org.matrix.msc2967.client:device:") %}
+                      {# <li>{{ scope }}</li> #}
```

Comment:
I recently added the `urn:mas:graphql:*` (= manage your account on MAS) and `urn:mas:admin` (= manage any account on MAS, when combined with the first one). Do you want to also add that here?

Reply:
Good shout. I've added some initial descriptions.